themes/theme-gmd/components/ProductFilters/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Small styling differences between the old and the new filter bar
![filters_1](https://user-images.githubusercontent.com/13087792/90135501-08fd0a80-dd73-11ea-8bb5-2ee3a74ef379.jpg)